### ISSUE TITLE
Error when executing the command ./looks-windows-amd64.exe generate

### ISSUE THREAD

**Issue by morismc09** [ClickPop/looks#21]
When I execute the .exe file without the argument "--no-meta", I get an index out of bound error. Maybe you can check the array length before you try to access the array in the routine.

```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.main()
        /home/runner/work/looks/looks/looks.go:33 +0x498
```

**Comment by morismc09** [comment]
At the moment the best workaround is to use a fake argument like test. When calling the .exe file with "generate test", it works.

**Comment by GV14982** [comment]
I just released v0.0.1, can you give that a try when you get a chance?

And an important thing is that you need to supply the generator with a config file so it knows where to find all your assets.

**Comment by GV14982** [comment]
I'm going to close this issue, feel free to submit another one if you're still having issues.
